pinax/stripe/actions/customers.py

```diff
@@ -45,12 +45,15 @@ def create(user, card=None, plan=settings.PINAX_STRIPE_DEFAULT_PLAN, charge_imme
     """
     trial_end = hooks.hookset.trial_period(user, plan)
 
-    stripe_customer = stripe.Customer.create(
-        email=user.email,
-        source=card,
-        plan=plan,
-        trial_end=trial_end
-    )
+    kwargs = {
+        'email': user.email,
+        'source': card,
+        'plan': plan,
+        'trial_end': trial_end,
+    }
+    if coupon:
+        kwargs.update({'coupon': coupon})
```

Comment:
agreed, although the main purpose of the PR is to support discount syncing on customer and subscription models.